main/guides/getting-started/Explainer - How to Start a Local Chain.md

```diff
@@ -0,0 +1,57 @@
+# Starting a Local Chain
+Starting a local chain using the Agoric SDK is easy! Just follow the steps outlined in this explainer. 
+As you're going through this explainer it may be helpful to watch this video walkthrough, showing you all the steps and the desired outcome of each.
+<iframe width="560" height="315" src="https://www.youtube.com/embed/XT1S4E1h-jM" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Installing Prerequisities:
+- Ensure Node is installed in your environment. If you need to install Node in your environment you can follow the [steps here](https://docs.agoric.com/guides/getting-started/#installing-nvm-and-node)
+- Ensure Yarn is installed in your environment. If you need to install Yarn in your environment you can follow the [steps here](https://docs.agoric.com/guides/getting-started/#installing-yarn)
+
+## Installing the Agoric SDK
+- Clone the Agoric SDK from the Agoric GitHub repo:
+ ```sh
+git clone https://github.com/Agoric/agoric-sdk
```

Comment:
We went through quite a bit of work to _avoid_ asking devs to clone the SDK. Let's please not go back.

Reply:
So if they have the docker container they don't need the SDK?

Reply:
Sort of.

In the `/getting-started/` flow, the `yarn install` step installs relevant parts of the SDK (where "relevant parts" is defined by the `package.json` file that results from `yarn create @agoric/dapp ...`).

Then, inside the container, there's another whole copy of the SDK.

Reply:
Okay, so to make sure I've got this correct, a user should:
- Download and start the Agoric docker container - `yarn start:docker`
- Create a new project using the command `yarn create @agoric/dapp --dapp-template dapp-offer-up myProject`
- Run `yarn install`
- Edit the dapp-offer-up smart contract to suit their needs
- Run `yarn start:contract`

Are these the correct steps?